packages/react-native-editor/CHANGELOG.md

```diff
@@ -1,37 +1,41 @@
-1.33.0
-------
+<!-- Learn how to maintain this file at https://github.com/WordPress/gutenberg/tree/master/packages#maintaining-changelogs. -->
+
```

Comment:
Let's also add a inline comment about the categorization each line has in front of it. Current format is:
```
[***] → Major new features, significant updates to core flows, or impactful fixes (e.g. a crash that impacts a lot of users) — things our users should be aware of.
[**] → Changes our users will probably notice, but doesn’t impact core flows. Most fixes.
[*] → Minor enhancements and fixes that address annoyances — things our users can miss.
```

Non userfacing fixes/features should use the `[*]` marker.

Also, note in the comment that all lines should have a categorization to make it clear when the author has missed categorizing the entry. Thoughts?

Reply:
Ah, while at it, please add that categorization guide to https://github.com/WordPress/gutenberg/tree/master/packages#maintaining-changelogs for better visibility.

Reply:
Do you think the remaining Gutenberg project package maintainers want to do that too? I think this is specific for GB-mobile no?

Reply:
> Do you think the remaining Gutenberg project package maintainers want to do that too?

Not really. Sorry, what I had in mind there was to have a native mobile specific section and add the categorization detail. All in all, someone following the link at the top of the changelog.md should eventually reach the same (categorization) info.

Reply:
I just added the categorization information as an inline comment on the top of the CHANGELOG file.

Reply:
Just to be sure, are you planning to add a mobile specific section in https://github.com/WordPress/gutenberg/tree/master/packages#maintaining-changelogs too? I still that'd be valuable.

Reply:
I wasn't because I think we shouldn't polute that file with a specific behaviour that is done in a single package. Maybe we could add an extra entry/file inside the package folder?

Reply:
Ah. It's not a single package though, right? We should have the same treatment for all mobile related packages, especially since its has it's own CHANGELOG.md file?

Reply:
I was not thinking to do it for react-native-aztec and react-native-bridge, changes there shouldn't have an effect on the user facing features, it's going to be more API changes that react-native-editor will use.

Reply:
Cool, we can introduce the same treatment there when/if needed.